jnius/jnius_jvm_dlopen.pxi

```diff
@@ -56,6 +56,19 @@ cdef find_java_home():
             except OSError:
                 break
         return dirname(dirname(java)).decode('utf8')
+    
+    if platform in ('darwin'):
```

Comment:
The parens do nothing here, this would match 'win' as well as 'darwin'. either put `(darwin', )` or just use `== 'darwin'`.